### ISSUE TITLE
Tutorial update 

### ISSUE THREAD

**Issue by rraallvv** [jellevdvliet/facebook-bot-autoresponder#3]
I really appreciate you sharing this tutorial, just wanted to point out a couple things.

In order to give the `manage_pages` and `publish_pages` permission Facebook requires that you send a screencast of showing off how the app works for someone in the review team to check the process.  I'm assuming the app won't work without those permissions, so there is a catch 22 situation here. Maybe Facebook hasn't realized yet they have an imposible requirement.

Also, they require specifying a platform, so I selected website, since it's the one that seems less complicated to setup, since other options like mobile app or Facebook game might require some other information that I don't have at hand.

For getting the Access Token once I've entered the website information, they require passing the website URL as a parameter and the `clientid` parameter was renamed to `clien_id`, but when I pass all that information I get the error shown bellow, even when I've already added the domain to the app settings:

>Can't load URL: The domain of this URL isn't included in the app's domains. To be able to load this URL, add all domains and sub-domains of your app to the App Domains field in your app settings.

Keep up the good work.

**Comment by gmjelle** [comment]
You can show a screencast with the bot commenting on your own (and/or app testers') comments, and explain in the app review what it does and that it basically runs from your own server/computer. However I don't know if FB would approve such an app these days, after the scandal in April. I haven't had the need yet to get the `manage_pages ` scope for an app, so I don't know how rigorous they are in approving apps these days. 

You should just select a website as the platform, indeed. More info in the server-to-server docs: https://developers.facebook.com/docs/apps/review/server-to-server-apps/

I'll have to check if the code still works, the API might have changed in the meantime.

**Comment by rraallvv** [comment]
@jellevdvliet Thanks. I'll check it out.

**Comment by rraallvv** [comment]
It seems facebook has their own docs outdated, the same thing happen with Google, the docs rarely shows the actual UI on the developer console. Interestingly enough I don't recall something like that happening when developing apps for iOS.

**Comment by rraallvv** [comment]
Here is how to get the long-lived token:

    curl -X GET "https://graph.facebook.com/oauth/access_token?
        grant_type=fb_exchange_token&
        client_id={app id}&
        client_secret={app secret}"

Source: https://developers.facebook.com/docs/facebook-login/access-tokens/refreshing/

Also I'm not sure if Facebook Login is needed, but I added it anyway as specified [here](https://github.com/facebook/php-graph-sdk/issues/963#issuecomment-375705042)

In short, needed to add the full callback url in the setting at
https://developers.facebook.com/apps/XXXXXXXXXXX/fb-login/settings/
replacing XXXXXXXXXXX with my app id

In the Facebook Login Settings then added the website address in the field Valid OAuth Redirect URIs
